src/components/button/index.stories.mdx

```diff
@@ -0,0 +1,43 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs';
+
+import Button from './index.tsx';
+
+<Meta title="Components/Button" component={Button} />
+
+export const Template = (args) => <Button {...args}>Primary button</Button>;
+
+# Button
+
```

Comment:
It's looking really nice!
It's going little by little in a lovely direction. Thank you so much for all the effort, patience and energy. 

I was thinking that it'd really help if we introduce all the component variations in the storyBook, as we know it should be a library. I was exploring some examples and we can find this -> https://style.monday.com/?path=/docs/buttons-button--sizes as a nice example. We can also introduce the Props table.
What do you think? is it make sense to you? I can also bring these two suggestions in the channel.

Reply:
Beautiful! 🙌🏼